vim25/soap/client.go

```diff
@@ -135,19 +135,34 @@ func ParseURL(s string) (*url.URL, error) {
 	return u, nil
 }
 
+// Go's ForceAttemptHTTP2 default is true, we disable by default.
+// This undocumented env var can be used to enable.
+var http2 = os.Getenv("GOVMOMI_HTTP2") == "true"
+
 func NewClient(u *url.URL, insecure bool) *Client {
 	var t *http.Transport
 
 	if d, ok := http.DefaultTransport.(*http.Transport); ok {
-		t = d.Clone()
+		// Inherit the same defaults explicitly set in http.DefaultTransport,
```

Comment:
skimming https://go.dev/src/net/http/transport.go's Clone() implementation it seems like we'd lose a few other defaults, right?

(would it make sense to clone and then override the ForceAttemptHTTP2 field only?)

Reply:
We're using the same defaults as http.DefaultTransport: https://github.com/golang/go/blob/8194d735cff90871b1ea5c92e83ddd50abdd4185/src/net/http/transport.go#L45-L56

And changing `ForceAttemptHTTP2=false`. See patch in the original issue, Clone() + ForceAttemptHTTP2=false breaks, unless `TLSClientConfig` set to nil. Rather than dig into this and what else could go wrong with Clone(), chose to revert to our original behavior of copying just the initial fields set by http.DefaultTransport.